### PR TITLE
Add sidebar for desktop layout on landing

### PR DESCRIPTION
**Pull request by meodai**
Introduces the possibility to add a sidebar on desktop.

`mainWindow` now comes with two more slots `slotSidebar` and `slotSidebarDecoration`. 

- `slotSidebar` holds the main content of the sidebar 
-  `slotSidebarDecoration` is used for the a visual of the dapps explorer

For now those only show marketing content, so they are completely hidden on mobile. This might change in the future if needed. 

Notable CSS changes:
- Add CSS for the sidebar itself: https://github.com/dfinity/internet-identity/pull/1705/files#diff-dfcfd73ef8243285e586ac8980bd197e24b434855418783109c0404283304b23R628-R664
- Put "design gutter" into a variable for easier reuse: https://github.com/dfinity/internet-identity/pull/1705/files#diff-dfcfd73ef8243285e586ac8980bd197e24b434855418783109c0404283304b23R138
- Add new reference colors and sizes for the sidebar https://github.com/dfinity/internet-identity/pull/1705/files#diff-dfcfd73ef8243285e586ac8980bd197e24b434855418783109c0404283304b23R171-R249
- Replace `min-width` and `max-width` both set to `40rem` by just `width`: https://github.com/dfinity/internet-identity/pull/1705/files#diff-dfcfd73ef8243285e586ac8980bd197e24b434855418783109c0404283304b23L625-R668
- Make sure the `.l-wrap` styles are not applied to `#pageContent` as well

Screenshot 2023-06-19 at 09 51 47

<hr/><details><summary>🟢 Some screens were added</summary></details><details><summary>🟡 Some screens were changed</summary></details><details><summary>🔴 Some screens were removed</summary></details>

### CODE REVIEW

File: screenshots/desktop/confirmSeedPhrase.png
Comment (nmattia) [inline screenshots/desktop/confirmSeedPhrase.png:1]: It looks like the whole page got narrower here, is that expected?

File: screenshots/desktop/landing_4.png
Comment (nmattia) [inline screenshots/desktop/landing_4.png:1]: I think there's a footer issue here; plus shouldn't there be a sidebar?

Comment (meodai) [reply screenshots/desktop/landing_4.png:1]: I think there is an issue with position fixed on the screenshot thingy, at least it works in real life.

**Comment by meodai** [comment]
replaced by #1732

**Pull request closed without merging**